force-app/main/default/classes/HierarchySettingsService.cls

```diff
@@ -0,0 +1,85 @@
+* @description Acts as the Service Layer between the HierarchySettingsChangesController and
+* the HierarchySettingsMapper.
+*/
+public virtual with sharing class HierarchySettingsService {
+    /**********************************************************************************
+    * @description Instance for Singleton Pattern
+    **********************************************************************************/
+    @TestVisible
+    private static HierarchySettingsService instance;
+
+    /**********************************************************************************
+    * @description Empty constructor is not public for Singleton Pattern
+    **********************************************************************************/
+    protected HierarchySettingsService() {}
+
+    /**********************************************************************************
+    * @description Static method to get the current instance for Singleton pattern
+    * @return The instance of HierarchySettingsService.
+    **********************************************************************************/
+    public static HierarchySettingsService getInstance() {
+        if (instance == null) {
+            instance = new HierarchySettingsService();
+        }
+
+        return instance;
+    }
+
+    /************************************************************************************************************
+    * @description Update Hierarchy Settings fields with specified changes.
+    * @param  hierarchySettingsChangesModel A HierarchySettingsChangesModel representing updates to Hierarchy Settings.
+    * @return True if the updates were completed successfully, false otherwise.
+    ************************************************************************************************************/ 
+    public virtual Boolean updateHierarchySettings(HierarchySettingsChangesModel hierarchySettingsChangesModel) {
+        return this.locateHierarchySettingsMapper().updateHierarchySettings(hierarchySettingsChangesModel);
+    }
+
+    /************************************************************************************************************
+    * @description Retrieve an instance of HierarchySettingsMapper
+    * @return An instance of the HierarchySettingsMapper class
+    ************************************************************************************************************/ 
+    @TestVisible
+    private HierarchySettingsMapper locateHierarchySettingsMapper() {
+        return HierarchySettingsMapper.getInstance();
+    }
+
+    /************************************************************************************************************
+    * @description Retrieve an instance of UserPermissionService
+    * @return An instance of the UserPermissionService class
+    ************************************************************************************************************/ 
+    // @TestVisible
```

Comment:
This is still commented out. Are we still waiting on that service? We can merge in the other branch and define this as a dependency for it.